temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -939,7 +956,9 @@ def __init__(self, *args, **kwargs) -> None:
     def __getattribute__(self, __name: str) -> Any:
```

Comment:
nit on existing code: double-underscore prefixes in Python should be (rarely) used, only when name-mangling is explicitly required. I don't think it's ever appropriate for a local variable in a function or a parameter name.

Reply:
Sure, I don't recall why I did this years ago, there might have been an issue with MyPy at that time